### ISSUE TITLE
Can't find the revision SHA in my github repo

### ISSUE THREAD

**Issue by philstraforelli** [STAT545-UBC/Discussion#323]
[On the webpage on how to submit homework](http://stat545.com/hw00_homework-guidelines.html#how-to-submit-homework), there is a direction that states: "Just above the file list, look for “latest commit” followed by ten numbers and letters (called the revision SHA) and a clipboard icon."

Here's what I see:
![](https://cloud.githubusercontent.com/assets/22040238/18622513/18e7a960-7de5-11e6-805e-e44dc3207128.PNG)

What follows "latest commit" is 7 characters long instead of 10, and I don't see a clipboard icon. Am I looking in the wrong location?

**Comment by jennybc** [comment]
FYI, the full SHA is 40 (!!!) characters long, but in many contexts only the first 7 characters are displayed.

You're right -- it seems the GitHub UI has changed since I wrote that, because I no longer see a clipboard icon there either.

Instead do this:

Just under "Private repository for the 2016 STAT 545 course work of bryan_jennifer", on the far left, click on "X commits".

There you will see your commits in reverse chronological order and the promised clipboard icon is there to load any SHA onto your clipboard. If you don't see that, it often means you need to update Flash.

**Comment by MayMAAhmed** [comment]
I was having the same problem.  After going to the far left and clicking my "x" commits, should I then only copy the latest commit from the clipboard icon?

**Comment by jennybc** [comment]
@MayMAAhmed Yes, they will be listed from most recent to oldest. You should copy the most recent one (or, in general, the one at which you consider your work to be "done").

TBH I generally look at the most recent version of things. But we are having you note the SHA also as an exercise for engaging with a bit of the Git and GitHub machinery.
